src/client/components/footer.js

```diff
@@ -31,50 +31,92 @@ function Footer(props) {
 
 	return (
 		<footer className="footer">
-			<Grid fluid>
-				<Row>
-					<Col xs={4}>
-						<small>{'Tested with '}
-							<a
-								href="https://www.browserstack.com/"
-								rel="noopener noreferrer"
-								target="_blank"
-							>
-								<img
-									alt="BrowserStack Logo"
-									height="25"
-									src="/images/BrowserStack.png"
-								/>
-							</a>
-						</small>
+			<Grid fluid className="padding-top-1 padding-bottom-1">
+				<Row >
+					<Col xs={6} xsOffset={3}>
+						<Row>
+							<Col xs={2} >
```

Comment:
For centering these items properly, here's the structure I ended up with:
<img width="402" alt="Capture d’écran 2021-01-13 à 18 19 44" src="https://user-images.githubusercontent.com/6179856/104486475-1c372f00-55cc-11eb-8c12-cef131a87f0f.png">

The modifications mainly require using `<Col xs={3} >`, since we have 4 elements and the bootstrap grid is arranged in 12 columns. With 4*3 col wide we use the full space of the parent element, while the parent's `xs={6} xsOffset={3}` ensures it is all centered.